src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -741,6 +747,51 @@ def delete_connectedk8s(cmd, client, resource_group_name, cluster_name,
     # Check Release Existance
     release_namespace = get_release_namespace(kube_config, kube_context, helm_client_location)
 
+    # Check forced delete flag
+    if(force_delete):
+
+        confirmation_message = "You can check using 'kubectl config get-contexts' to check if your current context is pointing to the the right cluster. \n" + "Are you sure you want to execute the delete command:"
```

Comment:
We need to refine this message a little. How about - "Please ensure you are pointing to the cluster that you intend to delete". @anraghun to suggest